phpBB/includes/acp/acp_storage.php

```diff
@@ -208,7 +215,7 @@ public function settings(string $id, string $mode)
 						{
 							$this->save_state();
 							meta_refresh(1, append_sid($this->u_action . '&amp;action=update&amp;hash=' . generate_link_hash('acp_storage')));
-							trigger_error($this->lang->lang('STORAGE_UPDATE_REDIRECT', $this->lang->lang('STORAGE_' . strtoupper($storage_name) . '_TITLE'), $i + 1, count($this->state['storages'])));
+							trigger_error($this->lang->lang('self::STORAGE_UPDATE_REDIRECT', $this->lang->lang('STORAGE_' . strtoupper($storage_name) . '_TITLE'), $i + 1, count($this->state['storages'])));
```

Comment:
Seems like this was changed by accident as I don't think we have this lang var.

Reply:
yes